### ISSUE TITLE
Invalid X-prop & activePoints supplied to label component in VictoryVoronoiContainer

### ISSUE THREAD

**Issue by jimMartin4321** [FormidableLabs/victory#1562]
# Bugs and Questions

### Checklist

- [x] This is not a `victory-native` specific issue. (Issues that only appear in `victory-native` should be opened [here](https://github.com/FormidableLabs/victory-native/issues/new))

- [x] I have read through the [FAQ](https://formidable.com/open-source/victory/docs/faq) and [Guides](https://formidable.com/open-source/victory/guides/) before asking a question

- [x] I am using the latest version of Victory

- [x] I've searched open issues to make sure I'm not opening a duplicate issue

### The Problem

When I set a custom x-domain it breaks the x-value that is passed to the VictoryVoronoiContainer label component. I have custom domains set for both x & y, however the y-prop is fine but the x-prop is wrong. It also consistently only activates the last data group.

### Reproduction

Here is a simple reproduction of the issue. If you hover over the chart and then open the console you can see that the y prop is correct but the x prop is off & the active point is always the same.
https://codesandbox.io/s/basic-victory-example-p3m19?file=/index.js

### A further note

I only implemented a custom x-domain today because after the update to 34.2.0 my x-domain is being set differently than it was in 34.1.3. Originally, the x-domain would autoscale to the minimum x-value and the maximum x-value. Now it goes from 0 to the maximum x-value. I am plotting against time stamps on the independent axis and since they are huge numbers all of the data is now shifted completely to the right. I did try to revert back to 34.1.3 for the time being but unfortunately since all of the sub dependencies for victory have a carat in front of them and the update is just a minor change I the sub deps always revert back to 34.2.0. Just wanted to bring this up as I saw another issue today that mentioned their scale breaking for brush container. [Link to the issue I am referencing](https://github.com/FormidableLabs/victory/issues/1560)

**Comment by boygirl** [comment]
@jimMartin4321 Thanks for the thourough issue!. I'm having a bit of a tough time with your reproduction since the data is undefined. The x value passed to the tooltip when you have voronoiDimension="x" set is meant to correspond to the position of the active point.  I will try instead to look into the underlying issue you are seeing with min domain no longer being set correctly for your data. Are you seeing this only with data on `VictoryGroup`?

**Comment by jimMartin4321** [comment]
@boygirl Thank you for responding so quickly!

My apologies, I am not very familiar with codesandbox.io, however I moved the data object above where it is mentioned and now the x-prop and active points are correct :homer:. So for that in my application there is most likely an issue w/ how data is being passed to my component, which I will investigate further on my end.

For the issue of the minimum domain, yes I am only seeing this with VictoryGroup. One of my components utilizes VictoryBar not nested in a victory group and the domain still works as expected.

**Comment by boygirl** [comment]
Thanks! That really helps narrow it down.

**Comment by jimMartin4321** [comment]
@boygirl alright so apologies I think I may have lead you a bit astray. I actually believe that the error has come from changes between 34.1.1 and 34.1.3. When I ran into this issue we updated from 34.1.1 directly to 34.2.0 but after going back I see the issue when we have 34.1.3 installed as well. I took a look at some the changes that took place between 34.1.1 & 34.1.3 and I found some code that looks like it could potentially be causing the issue.
https://github.com/FormidableLabs/victory/compare/v34.1.1...v34.1.3#diff-311d22492bf4fc376f64adef42f69ab0R22-R25
It looks like now if props.domain is set we set that to the domain for victory-group but for instance in my case I only have the domain set for y and not x. I obviously don't know all of your code base so this could not be the actual issue but it seemed related. Cheers and again thanks for the help.
